gdx/src/com/badlogic/gdx/utils/Array.java

```diff
@@ -459,6 +470,7 @@ public boolean equals (Object object) {
 		return true;
 	}
 
+	@Override

```

Comment:
You may have noticed from the source I'm a bit anal about the classes I maintain. I'd rather remove the `@Override`. Their use in library code is questionable, but their use on JDK classes that can never change is unnecessary.